pkg/cmd/root.go

```diff
@@ -39,6 +39,7 @@ var (
 		Long:  `A monitoring tool for PTP related metrics.`,
 		Run: func(cmd *cobra.Command, args []string) {
 			logging.SetupLogging(logLevel, os.Stdout)
+
```

Comment:
I'd prefer not to have unrelated whitespace changes like this, it makes the change set less clear.

Reply:
Sorry about that